<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1146</width>
    <height>649</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>GOQTtemplate3</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>724</x>
      <y>520</y>
      <width>91</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>打开图片</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>531</width>
      <height>441</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_OpenCam">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>560</y>
      <width>91</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>打开摄像头</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>561</width>
      <height>441</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_TakeAPic">
    <property name="geometry">
     <rect>
      <x>724</x>
      <y>560</y>
      <width>91</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>拍照</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_CloseCam">
    <property name="geometry">
     <rect>
      <x>524</x>
      <y>560</y>
      <width>91</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭摄像头</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Method">
    <property name="geometry">
     <rect>
      <x>864</x>
      <y>520</y>
      <width>91</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>算法</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Method_3">
    <property name="geometry">
     <rect>
      <x>1014</x>
      <y>560</y>
      <width>91</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboCamera">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>520</y>
      <width>149</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>364</x>
      <y>520</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>摄像头设备</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Method_2">
    <property name="geometry">
     <rect>
      <x>864</x>
      <y>560</y>
      <width>91</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>全屏</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>520</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>FPS:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>550</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>470</y>
      <width>131</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/new/prefix1/helu.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1146</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGoQTtemplate3">
    <property name="title">
     <string>GoQTtemplate3</string>
    </property>
   </widget>
   <addaction name="menuGoQTtemplate3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
